### PR TITLE
Added OpenShift command line instructions

### PR DESCRIPTION
**Pull request by sgilda**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -293,6 +295,107 @@ Use JBoss Developer Studio or Eclipse to Run the Quickstarts
...
+
+    Waiting for your DNS name to be available ... done
+
+    Cloning into 'APPLICATION_NAME'...
+    Warning: Permanently added the RSA host key for IP address '54.237.58.0' to the list of known hosts.
+
+    Your application 'APPLICATION_NAME' is now available.
+
+      URL:        http://APPLICATION_NAME-YOUR_DOMAIN_NAME.rhcloud.com/
+      SSH to:     52864af85973ca430200006f@APPLICATION_NAME-YOUR_DOMAIN_NAME.rhcloud.com
```
Comment (rafabene) [inline README.md]: The '52864af85973ca430200006f' should be a variable!

Comment (sgilda) [inline README.md]: Oops! Good catch! Thanks!

**Comment by sgilda** [comment]
@rafabene : Thanks for catching that. I replaced the UUID with a variable. Did you notice any other issues?

**Comment by rafabene** [comment]
It's good now to merge ;)

**Pull request merged**